### PR TITLE
feat(Call): add features to display self avatar or video and timer above video

### PR DESCRIPTION
**Pull request by MatveyNeyman**

### CODE REVIEW

File: src/components/Call/CallChat.js [hunk 1]
```diff
@@ -79,9 +81,19 @@ class CallChat extends PureComponent<Props, State> {
   }
 
   renderInfo() {
-    const { call, avatarSize } = this.props;
+    const { call, avatarSize, showTimerOnVideo } = this.props;
+
+    const isVideoCall = isOnCall(call.state) && hasTheirVideos(call);
+
+    if (isVideoCall) {
```
Comment (meafmira) [inline src/components/Call/CallChat.js]: Merge these two `if` in one

File: src/components/Call/types.js [hunk 2]
```diff
@@ -27,6 +28,8 @@ export type CallProps = {
   call: Call,
   avatarSize: number,
   isControlsDisabled?: boolean,
+  selfPeerInfo?: PeerInfo,
+  showTimerOnVideo?: boolean,
```
Comment (meafmira) [inline src/components/Call/types.js]: Remove optional and add default value

Comment (shabanov-dlg) [reply src/components/Call/types.js]: @meafmira default value for complex type may be a problem.

Sometimes implementation of this type can take a long time

Comment (meafmira) [reply src/components/Call/types.js]: I'm not about complex type. I'm about boolean

**Pull request merged**